patches/0008-src-Disable-unused-cpu-affinity-setting.patch

```diff
@@ -0,0 +1,136 @@
+From 81c5b790daa9599e94b5b3bc544a2e2e26cc7073 Mon Sep 17 00:00:00 2001
+From: Cezar Craciunoiu <cezar.craciunoiu@gmail.com>
+Date: Mon, 18 Jul 2022 16:21:02 +0300
+Subject: [PATCH 8/8] src: Disable unused cpu affinity setting
+
+Signed-off-by: Cezar Craciunoiu <cezar.craciunoiu@gmail.com>
+---
+ src/setcpuaffinity.c | 100 +------------------------------------------
+ 1 file changed, 2 insertions(+), 98 deletions(-)
+
+diff --git a/src/setcpuaffinity.c b/src/setcpuaffinity.c
+index 77b162103..c9d221adc 100644
+--- a/src/setcpuaffinity.c
++++ b/src/setcpuaffinity.c
+@@ -29,6 +29,8 @@
+ #include <stdlib.h>
+ #include <string.h>
+ #include <ctype.h>
++
++#define __linux__
```

Comment:
Would it not be better to simply delete the other `#ifdef`'s? Or maybe just place an `#undef` after the `#endif`? I know the result is the same, but IMHO it does not reflect the same intention that the upstream Redis code has.

For example, from what I can see, you did delete the `#ifdef`'s afterwards anyway.

Reply:
There's actually `ifdef` guards in `config.h` and that is included after the `ifdefs` you saw (check line 49)

Best I can do is `undef` the symbol afterwards, as it's not used anymore